nym-vpn-core/crates/nym-vpn-account-controller/src/commander.rs

```diff
@@ -45,7 +45,8 @@ impl AccountControllerCommander {
         self.store_account(mnemonic).await?;
         self.ensure_update_account().await?;
         self.ensure_update_device().await?;
-        self.ensure_register_device().await?;
+        // ignore registration errors
+        let _ = self.ensure_register_device().await?;
```

Comment:
We can just delete the line altogether. Once you store the account the registration is kicked off in the background. The `ensure_register_device()` call is for waiting for the result of this, but since we ignore the result we don't need to wait for it here